libs/cardano-ledger-core/src/Cardano/Ledger/Plutus/ExUnits.hs

```diff
@@ -32,6 +32,7 @@ import Cardano.Ledger.BaseTypes (
   BoundedRational (unboundRational),
   NonNegativeInterval,
   boundRational,
+  toRationalJSON,
```

Comment:
We no longer need to import it here, because this comment is no longer true:
https://github.com/IntersectMBO/cardano-ledger/blob/21871e9651d8d3fad3383647b4f157f1612ae685/libs/cardano-ledger-core/src/Cardano/Ledger/Plutus/ExUnits.hs#L172

So, we can fix the ToJSON/FromJSON instances for `Prices`, by relying on the underlying instance of `NonNegativeInterval`
```suggestion
```